nmstatectl/nmstatectl.py

```diff
@@ -44,6 +45,10 @@ def setup_subcommand_show(subparsers):
     parser_show.set_defaults(func=show)
     parser_show.add_argument('--yaml', help='Output as yaml', default=False,
                              action='store_true')
+    parser_show.add_argument(
+        '--only', default='*',
```

Comment:
We will probably have other entities (beyond interfaces), so how about just naming it "interfaces" or "filter-interfaces"?
"--interfaces" looks nice to me.

And we can use "all" instead of "*", sometimes "*" is reserved for other things in bash.

Reply:
I guess we could just use the same filter for the other entities as well. I would address this when there are other entities and the use-cases are clear. Since "*" is the default, it is not needed to specify `*` on the command-line, therefore I would keep it this way.

Reply:
Fair enough regarding the asterisk.

Regarding the name, this is a command line api, changing it later is problematic.
How would the user use --only to filter other things?
(Interesting enough, it the last few days several "other" things came up frequently)

Reply:
I assume the other entities would have names as well and --only would only the entities for which the pattern matches, since interfaces are the only entities currently, it can only match interfaces.

Reply:
Yes, they will probably have a name or some other id, but also a "context" (interfaces).
Can we use --only with another argument that gives the context as well? Maybe something crazy like: --only interfaces:a,b,c,d
Sorry for nagging on this, but it feels too generic and we will have another tree branch pretty soon.